pytracking/tracker/base/basetracker.py

```diff
@@ -52,6 +52,60 @@ def track_sequence(self, sequence):
 
         return tracked_bb, times
 
+    def track_videofile(self,videofilepath):
```

Comment:
Would be nice to have an optional bbox parameter so that it's possible to input the target bounding box, instead of always selecting it. If the no bbox is provided, the user is asked to draw the bounding box